test/fixtures/server.js

```diff
@@ -127,6 +127,7 @@ export async function createServer() {
                   try {
                     await writer.ready
                     await writer.write(Uint8Array.from([0, 1, 2, 3, 4]))
+                    await new Promise((resolve) => setTimeout(resolve, 50)) // do not flood everything
```

Comment:
I don't know that this is a great solution - adding arbitrary timeouts in the test code implies there are race conditions hiding in the production code.

Furthermore it requires the sending peer to "do the right thing" which is almost certainly not to be the case in the real world - the receiving peer should be able to handle anything the client throws at it in order to project itself.

Reply:
This is just a rate limitation. Without the delay, you are filling the client's buffer so full, that the client does not recover.  And it's actually unrealistic to flood with a network bandwidth available on the same pc. This is a substantial ddos attack in fact. And the problem is on the client side, it is actually an issue in chromium, so if something needs to be fixed it is on their side.

Reply:
I have posted it as a bug to chromium:
https://bugs.chromium.org/p/chromium/issues/detail?id=1408176
may be the developer there can help.

Reply:
Now it is crashing even with the change in the CI/CD.  It is really problematic, that it is so sporadic. But I still think it is caused by the datagrams and surfaces in the tests after the datagram test.